site/docs/foundations/data-visualization/index.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Chart types
+layout: DetailTechnical
+sidebar:
+  groupLabel: Data visualization
+  label: Chart types
+  groupPriority: 10
+---
+
+Data visualization presents information in a visual form that’s easy to understand. When applied to dense and complex datasets, it aids in highlighting trends and patterns.
+
+## Chart selection
+
+Choosing the best chart for your use case depends on:
```

Comment:
Can we provide a brief written example for each of these points? They're somewhat vague atm. Would someone understand what each point means?